File: Nível 3/1171 - Frequência de Números/main.py
quantia = int(input())
contador = {}

for i in range(1, quantia + 1):
    elemento = int(input())
    if elemento in contador:
        contador[elemento] += 1
    else:
        contador[elemento] = 1

for elemento in sorted(contador):
    print(f"{elemento} aparece {contador[elemento]} vez(es)")

File: Nível 3/1171 - Frequência de Números/main.lua
contador = {}
quantia = tonumber(io.read())

for i = 1, quantia do
    elemento = tonumber(io.read())
    if contador[elemento] then
        contador[elemento] = contador[elemento] + 1
    else
        contador[elemento] = 1
    end
end

ordenado = {}
for elemento in pairs(contador) do
    table.insert(ordenado, elemento)
end
table.sort(ordenado)

for _, chave in ipairs(ordenado) do
    print(string.format("%s aparece %d vez(es)", chave, contador[chave]))
end